main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"./migraph"
```

Comment:
@krnowak thanks! Also I noticed there is sometimes a empty line after the package names starting with `github.com/...`, should there be one here as well?